docs/user/check-commands.rst

```diff
@@ -1552,6 +1552,65 @@ The ``eventlog()`` function allows you to conveniently count EventLog_ events by
 
     The ``count()`` method internally requests the EventLog Viewer's "count" JSON endpoint.
 
+AppDynamics
+-------------
+
+Enable AppDynamics Healthrule violations check.
+
+.. py:function:: appdynamics(url)
+
+Methods of AppDynamics
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. py:function:: healthrule_violations(application, time_range_type='', duration_in_mins=None, start_time=None, end_time=None, severity=None)
+
+    ::
+
+    Return Healthrule violations for AppDynamics application.
+
+    :param application: Application name or ID
+    :type application: str
+
+    :param time_range_type: Valid time range type.
+    :type time_range_type: str
+
+    :param duration_in_mins: Time duration in mins. Required for BEFORE_NOW, AFTER_TIME, BEFORE_TIME range types.
+    :type duration_in_mins: int
+
+    :param start_time: Start time (in milliseconds) from which the metric data is returned.
```

Comment:
There are no defaults for start/end times, since AppDyanmics API require this input explicitly.